x-pack/platform/test/functional/apps/aiops/cases/change_point_detection_cases.ts

```diff
@@ -5,8 +5,8 @@
  * 2.0.
  */
 
-import type { FtrProviderContext } from '../../ftr_provider_context';
-import { USER } from '../../../api_integration/services/ml/security_common';
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../api_integration/services/ml/security_common';
```

Comment:
Note when backporting to 8.19, there will be a `skip` to remove from the test suite in this file.